scripts/ci/ci_build_dockerhub.sh

```diff
@@ -51,12 +51,12 @@ else
    echo "DOCKER_TAG=${DOCKER_TAG}"
 fi
 
-[[ ${DOCKER_TAG:=} =~ ${DEFAULT_BRANCH}-python([0-9.]*)(.*) ]] && export PYTHON_MAJOR_MINOR_VERSION=${BASH_REMATCH[1]}
+[[ ${DOCKER_TAG:=} =~ .*-python([0-9.]*)(.*) ]] && export PYTHON_MAJOR_MINOR_VERSION=${BASH_REMATCH[1]}
```

Comment:
Why do we need to remove `${DEFAULT_BRANCH}`

Reply:
It was not needed. It's remnant from the old script and ${DEFAULT_BRANCH} was actually empty :)

Reply:
This way in the future we can have other branches as well and we will also be able to use the scripts for automated "tag" release building as well.